CHANGES.rst

```diff
@@ -4,6 +4,12 @@ Latest changes
 In development
 --------------
 
+- Make `joblib.hash` and `joblib.Memory` caching system compatible with `numpy
+  >= 1.20.0`.  In the future, users updating `joblib` to a release that
```

Comment:
I should include "joblib hash values are now not guaranteed to stay the same across joblib versions". Waiting for CI to finish before adding this.